docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
@LalatenduMohanty It says 'restart' today because that is the only verb supported in vsm.  @budhrg is working on a patch that will extend my PR VSM/#218 to add 'start' as a verb.  When that merges, this PR will update to read 'start' and not 'restart'.  You made a comment, that seemed accidental, about Vagrantfiles that was confusing.